sdks/python/apache_beam/runners/dataflow/internal/clients/dataflow/message_matchers.py

```diff
@@ -49,7 +51,7 @@ def _matches(self, item):
     if self.origin != IGNORED and item.origin != self.origin:
       return False
     if self.context != IGNORED:
-      for key, name in self.context.iteritems():
+      for key, name in iteritems(self.context):
```

Comment:
Why `iteritems` here vs `.items()`?

Reply:
To avoid efficiency losses in Python2, see: https://github.com/apache/beam/pull/5373#discussion_r195550396